subcommittees/mentoring/PracticumProposal.md

```diff
@@ -0,0 +1,192 @@
+connections which may lead to more discussions between instructors at a later stage. Also, by having
+people already within the organization mentoring future instructors, they get to know people already
+inside of the organization. 
+
+Fourth, as a bonus we get a lot of people working through the material, which can only be a benefit to
+the lesson development process. It might draw more people into helping out with lesson development, and
+could lower the bar for people developing new lessons.
+
+With these goals in mind, we have developed a post-instructor training practicum.  
+
+## Practicum elements
+
+The instructor training practicum aims to facilitate the transition from the introductory instructor
+training  to actually teaching workshops, by doing the following activities: 
+
+### Mentoring
+
+Future instructors, or trainees, form small groups and are assigned a mentor.  Over a period of a 3-4
+weeks, they work through the core material* for Software and Data Carpentry on their own, as if they
+were workshop participants.  Each week, the group meets to discuss their experiences with the lesson
+material and the mentor receives feedback and provides guidance.  The primary goal is not that the
+trainees should be prepared enough to teach the module, but to become familiar with it and gain skills.  
+
+### Mini-lessons
+
+Each trainee chooses a topic relevant to Software and Data Carpentry (can be one of our lessons or
+their own provided their mentor approves) and teaches a 1 hour lesson at their local institution to an
+audience of their choice.  This lesson should be at least partially observed by a mentor and feedback
+provided.  Whenever possible, the mentor observing the mini-lesson should be the mentor of the previous
+mentoring element.  
+
+### General comments
+
+These two elements need not happen consecutively, but in the first implementation of this practicum
+they will be consecutive, not parallel (see timeline).  
+
+These two elements will be required to become a fully badged Software or Data Carpentry Instructor.  
+
+## Implementation Logistics
+
+### Mentoring Groups
+
+For the first practicum, all recent participants in instructor training (online 8 week, December 2 day)
+will be invited to join.  This program may be extended to include all newly trained instructors after
+we have figured out how it works.
+
+Candidates will sign up via a form that requests the following information: 
+
+- Wants to teach Software Carpentry and/or Data Carpentry 
+- Whether they want to do Python or R  
+- group member preference from their training (if any) 
+- time zone + geographic location (nearest airport), preferred time of day for meetings (daytime/evening) 
+- Language preferences (if any). 
+- When they went through instructor training 
+ 
+Based on this data, the mentoring committee will form groups of 3-4 people, prioritizing in the order
+listed above.  
+
+### Mentors
```

Comment:
On Sat, Nov 28, 2015 at 10:58:26AM -0800, Katy Huff wrote:
> If I saw a request for 6-10 hours of my time over the next month…

I think it's a bit more than that.  Picking the minimums of each range
(three learners in a group, three weeks for the mentoring phase, 2
hours a week of email mentoring), that gives:

  (1 hour online per week + 2 hours email per week) * 3 weeks +
  2 hours of mini-lesson work per learner * 3 learners

that's going to be 15 hours in the month.  At the high side of each
range it's:

  (1 + 3) * 4 + 2 * 4 = 24 hours

So that's at least as much time commitment as teaching a regular
workshop.  We seem to have enough volunteers for that, so maybe count
this along with regular instruction for SCF membership [1]?  I'm not
sure what a “senior instructor” badge would mean [2] (and I have vague
memories of previous discussions that ended up rejecting formal
differentiation within the instructor pool).  And I'm not sure how
popular a “thanks for donating a sizable chunk of time, we may give
you preferential treatment if you donate another sizable chunk of time
this year [3]” offer would be.

[1]: https://github.com/swcarpentry/board/blob/752917bd247cf4456a03b24b7cc18df3c777e093/governance.md#membership
[2]: https://github.com/swcarpentry/board/pull/65/files#r46095658
[3]: https://github.com/swcarpentry/board/pull/65/files#r46102260